public/js/selfoss-ui.js

```diff
@@ -71,6 +71,53 @@ selfoss.ui = {
                     $('.mark-these-read').hide();
             }
         }
+    },
+
+
+    /**
+     * show error
+     *
+     * @return void
+     * @param message string
+     */
+    showError: function(message) {
+        selfoss.ui.showMessage(message, 'undefined', 'undefined', true);
```

Comment:
This should probably be quoteless, since `typeof 'undefined' === 'string'`, leading to showing button with undefined text in error messages (e.g. invalid fragment like `#invalid`)